frontend/src/app/main/data/workspace.cljs

```diff
@@ -326,7 +326,10 @@
   (ptk/reify ::delete-page
     ptk/WatchEvent
     (watch [it state _]
-      (let [page (get-in state [:workspace-data :pages-index id])
+      (let [pages (get-in state [:workspace-data :pages])
+            page-num (d/index-of pages id)
+            page (get-in state [:workspace-data :pages-index id])
+            page (assoc page :page-num page-num)
```

Comment:
I think is better call it `:index` on page data sturcture

Reply:
fixed